Client/Frontend/Browser/BrowserViewController/BrowserViewController+FindInPage.swift

```diff
@@ -26,9 +26,10 @@ extension BrowserViewController {
             self.view.layoutIfNeeded()
         })
 
-        findInPageBar.snp.makeConstraints { make in
-            make.height.equalTo(UIConstants.ToolbarHeight)
-        }
+        findInPageBar.heightAnchor.constraint(
+            greaterThanOrEqualToConstant: UIConstants.ToolbarHeight
```

Comment:
Nice change: setting the constraint greaterThanOrEqual. This helps support Dynamic Text, but in order to be responsive, you need to add adjustsFontForContentSizeCategory = true to the searchText and matchCountView.